### ISSUE TITLE
Perhaps consider protecting user

### ISSUE THREAD

**Issue by kootenpv** [jlesquembre/autopilot#5]
I'm thinking perhaps it fits in your framework to have an option to enable data protection, e.g. have a look here: https://github.com/kootenpv/gittyleaks

Goal would be to prevent commits/uploads with bad data.

**Comment by jlesquembre** [comment]
Yes, I like the idea. I was thinking about doing some checks before a new release. I thought about integrating [pyroma](https://pypi.python.org/pypi/pyroma) and [chardet](https://pypi.python.org/pypi/chardet), but I also like the idea of integrate your project.

But for now it has to wait, before I want to focus on having a more stable version with the basic features. I'm keeping the issue open to not forget about it.
